crowd_anki/representation/note.py

```diff
@@ -116,12 +117,23 @@ def save_to_collection(self, collection, deck, model_map_cache):
         else:
             self.handle_model_update(collection, model_map_cache)
 
+        self.handle_import_config_changes(import_config, note_model)
+
         self.anki_object.__dict__.update(self.anki_object_dict)
         self.anki_object.mid = note_model.anki_dict["id"]
         self.anki_object.mod = anki.utils.intTime()
         self.anki_object.flush()
 
         if new_note:
             collection.addNote(self.anki_object)
-        elif not self.config.import_notes_ignore_deck_movement:
+        elif not import_config.ignore_deck_movement:
             self.move_cards_to_deck(deck.anki_dict["id"])
+
+    def handle_import_config_changes(self, import_config, note_model):
+        # Personal Fields
+        for num, field in enumerate(self.anki_object_dict["fields"]):
```

Comment:
nit: the field var is never used, rename to _ (which is usually used for "throwaway vars")